src/routes/entry-point.ts

```diff
@@ -1,23 +1,28 @@
+import { constants } from 'http2';
 import { Next } from 'koa';
-import { hydra, schema } from 'rdf-namespaces';
+import { addAll } from 'rdf-dataset-ext';
+import url from 'url';
 import { AppContext, AppMiddleware } from '../app';
+import { hydra, rdf, schema } from '../namespaces';
 import Routes from './index';
 
 export default (): AppMiddleware => (
-  async ({ request, response, router }: AppContext, next: Next): Promise<void> => {
-    response.body = {
-      '@context': {
-        '@base': request.origin,
-      },
-      '@id': router.url(Routes.EntryPoint),
-      '@type': schema.EntryPoint,
-      [schema.name]: { '@value': 'Article Store', '@language': 'en' },
-      [hydra.collection]: {
-        '@id': router.url(Routes.ArticleList),
-        '@type': hydra.Collection,
-      },
-    };
-    response.type = 'jsonld';
+  async ({
+    dataFactory: { quad, literal, namedNode }, request, response, router,
+  }: AppContext, next: Next): Promise<void> => {
+    const articleList = namedNode(url.resolve(request.origin, router.url(Routes.ArticleList)));
+    const entryPoint = namedNode(url.resolve(request.origin, router.url(Routes.EntryPoint)));
```

Comment:
It'd be better for all these named nodes to be relative, then make absolute when writing the response body.